Source/WebInspectorUI/UserInterface/Views/ConsolePrompt.js

```diff
@@ -267,6 +267,8 @@ WI.ConsolePrompt = class ConsolePrompt extends WI.View
 
             if (this.delegate && typeof this.delegate.consolePromptTextCommitted === "function")
                 this.delegate.consolePromptTextCommitted(this, currentText);
+
+            WI.javaScriptRuntimeCompletionProvider.clearCache();
```

Comment:
this is not the right place to put this, as currently `WI.ConsolePrompt` is used for more than just JavaScript (e.g. WebSQL)

i think this should be moved to `WI.JavaScriptLogViewController.prototype.consolePromptTextCommitted`